tests/molecules-cells/pass_ms.js

```diff
@@ -0,0 +1,33 @@
+const complements = [
+  ['G', 'C'],
+  ['C', 'G'],
+  ['T', 'A'],
+  ['A', 'U'],
+]
+
+const RNA = (dna) => {
+  let res = ''
+  let obj = Object.fromEntries(complements)
+  for (let i of dna.split('')) {
+    res += obj[i]
+  }
+  return res
+}
+
+const DNA = (rna) => {
+  let res = ''
+  let invobj = invert(Object.fromEntries(complements))
+  for (let i of rna.split('')) {
+    res += invobj[i]
+  }
+  return res
+}
+
+const invert = (obj) => {
+  let ret = {}
+  for (let key of Object.keys(obj)) {
+    ret[obj[key]] = key
+  }
+  obj = ret
+  return ret
+}
```

Comment:
You should try to refactor your solution a bit.
It's good to have variance in solution, but I think it would be a good exercise for you @MSilva95.
I like your idea to have a your dna data matches in an entry instead of objects to use the same for both.

but then your invert function is quite complexe, which kind of cancel out the benefits, and both of your functions are almost exactly the same.

It's not that it's bad but more so than in go, JS allows to avoid a lot of repetition.

Here my exercise for you:
 - no for loops
 - no logic duplication